pkgs/by-name/co/copilot-language-server/package.nix

```diff
@@ -36,18 +52,24 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     stripRoot = false;
   };
 
-  npmDepsHash = "sha256-PLX/mN7xu8gMh2BkkyTncP3+rJ3nBmX+pHxl0ONXbe4=";
```

Comment:
Okay I wasn't crazy... didn't think this was used when doing the bump lol

Reply:
I think it just went under the radar in the code changes introduced in #388201.
Should I make a new commit for such a change?

Reply:
Nah, it's fine